game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadFileProperties.java

```diff
@@ -51,7 +51,9 @@ private static File fromZip(final File zipFile) {
     return new File(zipFile.getAbsolutePath() + ".properties");
   }
 
-  public Optional<Version> getVersion() {
-    return Optional.ofNullable(props.getProperty(VERSION_PROPERTY)).map(Version::new);
+  public Optional<Integer> getVersion() {
+    return Optional.ofNullable(props.getProperty(VERSION_PROPERTY))
```

Comment:
Doesn't `props.getProperty(VERSION_PROPERTY)` return an integer now?
Or am I just missing something?

Reply:
It is still a string. It is actually required to store it as a string, storing an Integer throws an IllegalArgumentException.